modules/shortcodes/vr.php

```diff
@@ -129,3 +128,29 @@ function jetpack_vr_viewer_shortcode( $atts ) {
 }
 
 add_shortcode( 'vr', 'jetpack_vr_viewer_shortcode' );
+
+// Gutenberg!
+add_action( 'admin_init', 'jetpack_register_block_type_vr' );
+function jetpack_register_block_type_vr() {
+	if ( ! function_exists( 'register_block_type' ) ) {
+		return;
+	}
+
+	wp_register_script(
+		'jetpack_vr_viewer_shortcode_editor_script',
+		Jetpack::get_file_url_for_environment( '_inc/build/shortcodes/js/blocks/vr-block.min.js', 'modules/shortcodes/js/blocks/vr-block.js' ),
+		array( 'wp-blocks', 'wp-element' )
+	);
+
+	wp_register_style(
+		'jetpack_vr_viewer_shortcode_editor_style',
+		plugins_url( 'modules/shortcodes/css/blocks/vr-block.css', JETPACK__PLUGIN_FILE ),
```

Comment:
I don't know Jetpack code, but it seems like CSS could also use help from `Jetpack::get_file_url_for_environment` util 🤷‍♂️

Reply:
Alas this is just an inconsistency in Jetpack - we minify JS, but we _concatenate_ CSS, but not _all_ CSS... anyway, this just just something where I'll leave it decoupled and we can worry about minifying the CSS and using that helper function later.